app/src/main/java/com/example/instagramclone/fragments/main_screen/compose/StoriesSection.kt

```diff
@@ -0,0 +1,20 @@
+package com.example.instagramclone.fragments.main_screen.compose
+
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyRow
+import androidx.compose.foundation.lazy.items
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.unit.dp
+import com.example.domain.entities.Post
+
+@Composable
+fun StoriesSection(postList: List<Post>) {
+    LazyRow(modifier = Modifier.padding(vertical = 10.dp)) {
```

Comment:
It's best to use the [4-point grid system](https://medium.com/@aratidube12lns/4-point-grid-system-for-more-consistent-interface-design-efea81dea3f3#:~:text=What%20is%20the%204%2Dpoint,padding%2C%20margins%20and%20line%20heights.) wherever possible.